lab-sarah/README.md

```diff
@@ -0,0 +1,52 @@
+# Project description
+
+Creates a RESTful multiple-resource API with a persistence layer powered by Mongo. User's can sign-up, login, and make changes to a a gallery using CRUD operations.
+
+## How to install
+
+See package.json for list of dependencies necessary to install and run this project.
+
+## How to start the server
+
+From the root level of the project, enter 'npm start' in terminal to start the server.
+
+## Server endpoints
+
+### authRouter
+
+POST requests
+
+/api/signup
+
+* Create an 'account' with a unique username, password, and email address.
```

Comment:
You should tell the client how to structure their request body -- what the key value pairs and required data types are.